app/components/buttons/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -0,0 +1,30 @@
+import style from "./ToggleSwitch.module.css";
+import { Tooltip } from "shared";
+import { classNames } from "pi-ui";
+
+// enabled shows the switch as on or off.
+// disabled unable to trigger the onClick method.
+// enabledText and notEnabledText the tooltop contents when the switch is on/off.
+const ToggleSwitch = ({ enabled, onClick, disabled, enabledText, notEnabledText }) => (
+  <Tooltip
+    text={ enabled ? enabledText : notEnabledText }>
+    <div className={classNames(style.toggleSwitch)}>
+      <div
+        className={classNames(
+          enabled
```

Comment:
actually here you don't need the classNames, it's useful when you have something like
`classNames(isOdd && style.odd, isGreen && style.green, ...` but in your case the condition will produce one class either enabled or disabled, so it can be simplified to not use classNames